### PR TITLE
feat: Doctrine

### PR DESCRIPTION
**Pull request by ntsanq**
this is function for using doctrine

### CODE REVIEW

File: templates/home/key.html.twig [hunk 1]
```diff
@@ -0,0 +1 @@
+your key is {{ s3key }}
```
Comment (khajackie2206) [inline templates/home/key.html.twig:1]: EOL

Comment (ntsanq) [reply templates/home/key.html.twig:1]: Thank you

File: src/Repository/CarRepository.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    {
+        $this->getEntityManager()->remove($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+//    /**
+//     * @return Car[] Returns an array of Car objects
```
[the remark this answers is not in the document]
Comment (ntsanq) [reply src/Repository/CarRepository.php:43]: Okay I will remove that

File: src/Repository/UserRepository.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    public function remove(User $entity, bool $flush = false): void
+    {
+        $this->getEntityManager()->remove($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+//    /**
```
[the remark this answers is not in the document]
Comment (ntsanq) [reply src/Repository/UserRepository.php:42]: yeah, thank you for that

**Pull request closed without merging**